config/config.go

```diff
@@ -25,21 +23,14 @@ func LoadConfiguration() *viper.Viper {
 	config.Token = v.GetString("CONSUL_TOKEN")
 
 	c, err := api.NewClient(config)
-	PanicOnError(err, "")
+	utils.PanicOnError(err, "")
 
 	pair, _, err := c.KV().Get(v.GetString("CONSUL_PATH"), nil)
-	PanicOnError(err, "Invalid consul config.")
+	utils.PanicOnError(err, "Invalid consul config.")
```

Comment:
yeah perhaps a bit overkill, but I'll opt for the 1-liner most of the time so will be strewn across everything I work on. may as well keep it here